smarts/core/trap_manager.py

```diff
@@ -145,6 +145,10 @@ def largest_vehicle_plane_dimension(vehicle):
                 ),
             )
             for v_id in sorted_vehicle_ids:
+                # Skip the capturing process if history traffic is used
+                if sim.scenario.traffic_history:
```

Comment:
One assumption here is when doing imitation learning, there will be no SUMO traffic, therefore we wanted to disable the trapping.